### PR TITLE
Slot labels and slot switching

### PR DESCRIPTION
**Pull request by Elinvynia**
Now with comments :)

https://user-images.githubusercontent.com/59487684/104851229-1b680b00-58f4-11eb-805d-e6361867217c.mp4

### CODE REVIEW

File: enfaria-game/src/ui/inventory.gd [hunk 1]
```diff
@@ -51,6 +62,50 @@ func _process(_delta):
...
+        var label = Label.new()
+        label.add_color_override("font_color", Color(Color.black))
+        label.text = str(num)
+        label.margin_top = 1
+        label.margin_left = 1
+        num += 1
+        slot.add_child(label)
+
+
+func handle_switch(event):
```
Comment (meesha7) [inline enfaria-game/src/ui/inventory.gd:86]: I think there might be a better way, I'm not sure but this usually can be done more dynamically. What do you think?

**Review by meesha7 — APPROVED** [review]
It gets better and better with every PR you make ;)

File: enfaria-game/src/ui/inventory.gd (hunk 1 above)
Comment (Elinvynia) [reply enfaria-game/src/ui/inventory.gd:86]: I looked into it and it looks like there is no way to extract the action from an event due to the way they are represented internally, we can only check if it is a certain action (due to the inputmap and scandoes)

Comment (meesha7) [reply enfaria-game/src/ui/inventory.gd:86]: Thanks for the explanation

**Pull request merged**